avocado/core/runner.py

```diff
@@ -513,11 +514,13 @@ def run_suite(self, test_suite, mux, timeout=0, replay_map=None):
         no_digits = len(str(test_result_total))
         self.result.tests_total = test_result_total
         self.result.start_tests()
+        tests_tmpdir = teststmpdir.create()
```

Comment:
Viz: https://github.com/avocado-framework/avocado/pull/1671/commits/b93c02b7d81c99a10526bba15f115ffa0d9836d1#r92342965 I don't think we should share `teststmpdir` across all runners in one avocado execution. I'd simply use:

```
from . import teststmpdir
tests_tmpdir = teststmpdir.teststmpdir("AVOCADO_TESTS_TMPDIR")
tests_tmpdir.create()
```